src/components/absorber/module.cairo

```diff
@@ -439,4 +440,63 @@ mod Test {
         let ton_equivalent = 0;
         Absorber::AbsorberImpl::set_absorptions(ref state, 0, times, absorptions, ton_equivalent);
     }
+
+    use alexandria_storage::list::{List, ListTrait};
+
+    #[starknet::interface]
+    trait ITestList<T> {
```

Comment:
It's strange to have code for test and with `Test` name in code and not in Test file nether in test mod ?
Does it need to be place in other place ?